cns/client_test.go

```diff
@@ -1182,7 +1234,7 @@ func isvSphereVersion70U3orAbove(ctx context.Context, aboutInfo types.AboutInfo)
 	version := strings.Join(items[:], "")
 	// Convert version string to string, Ex: "7.0.3" becomes 703, "7.0.3.1" becomes 703
 	if len(version) >= 3 {
-		vSphereVersionInt, err := strconv.Atoi(version[0:2])
+		vSphereVersionInt, err := strconv.Atoi(version[0:3])
```

Comment:
For 6.7u3, the aboutInfo.Version is 6.7.0, i.e., only 3 digits. Can we keep this same as before ?

Reply:
`version[0:2]` only take the first two elements of the version array. Basically, using `version[0:2]`, '6.7.0' -> 67 and '7.0.3' -> 70. I don't think it is the expected behavior.

Reply:
Makes sense. My bad, I thought it is inclusive.